packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,942 @@
+        <Option value="put">PUT</Option>
+      </Select>
+    </Box>
+  </Do>
+  <Dont
+    center
+    title="Don't"
+    body="Don’t use a disabled form field to show information that can’t be edited. Instead, use a read-only form field or consider another way of showing static information."
+  >
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="disabled_dont">Company</FormLabel>
+      <Select disabled id="disabled_dont" name="disabled_dont" onChange="">
+        <Option value="get">Twilio</Option>
+      </Select>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do center title="Do" body="Use a checkbox group to allow users to make multiple selections.">
+  <Box width="100%" padding="space60">
+     <CheckboxGroup
+      name="multiple_do"
+      legend="Select the clients you would like to test."
+    >
+      <Checkbox id="apple_mail" value="apple_mail">
+        Apple Mail
+      </Checkbox>
+      <Checkbox id="g_mail" value="g_mail">
+        GMail
+      </Checkbox>
+      <Checkbox id="outlook" value="outlook">
+        Outlook
+      </Checkbox>
+    </CheckboxGroup>
+  </Box>
+  </Do>
+  <Dont
+    center
+    title="Don't"
+    body="When possible, avoid allowing multiple selection on a select component."
+  >
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="multiple_dont">Select the clients you would like to test</FormLabel>
+      <Select multiple size="3" id="multiple_dont" name="multiple_dont" onChange="">
+        <Option value="apple_mail">Apple Mail</Option>
+        <Option value="g_mail">GMail</Option>
+        <Option value="outlook">Outlook</Option>
+      </Select>
+    </Box>
+  </Dont>
+</DoDont>
+
+## Anatomy
+
+<table>
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Default token</th>
+      <th>Modifiable?</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Label text</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>$font-size-30</ListItem>
+          <ListItem>Default: $color-text, $font-weight-semibold</ListItem>
+          <ListItem>Disabled: $color-text-weak, $font-weight-semibold</ListItem>
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Required field indicator</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>4px size (0.25rem)</ListItem>
+          <ListItem>Background: $color-background-required</ListItem>
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Box Shadow</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>Default: $shadow-border-input</ListItem>
+          <ListItem>Hover: shadow-border-inputHover</ListItem>
+          <ListItem>Disabled: shadow-border-input-disabled</ListItem>
+          <ListItem>Error: shadow-border-input-disabled</ListItem>
+          <ListItem>Error hover: shadow-border-input-error-hover</ListItem>
```

Comment:
These are different now based on PR #425 . I think you should be able to copy the anatomy section in Input